sdk/quantum/ci.yml

```diff
@@ -0,0 +1,30 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+    branches:
+        include:
+            - master
+            - hotfix/*
+            - release/*
+    paths:
+        include:
+            - sdk/quantum/
```

Comment:
Will other libraries, such as management plane, be added into the service directory? This include is very generic and may cause pipelines to run when they shouldn't. We should scope the include paths to code paths that should trigger the client SDKs to build, here is an example:

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/ci.yml